common/src/main/java/tachyon/security/authorization/PermissionStatus.java

```diff
@@ -46,8 +52,13 @@ public FsPermission getPermission() {
    * @see FsPermission#applyUMask(FsPermission)
    */
   public PermissionStatus applyUMask(FsPermission umask) {
-    mPermission = mPermission.applyUMask(umask);
-    return this;
+    FsPermission newFsPermission = mPermission.applyUMask(umask);
+    return new PermissionStatus(mUsername, mGroupname, newFsPermission);
+  }
+
+  /** Get the Directory default PermissionStatus. */
+  public static PermissionStatus getDirDefault() {
+    return new PermissionStatus("", "", new FsPermission((short)0755));
```

Comment:
Hi @ooq , I use the constants configuration. I think it doesn't need to change it configurable. The default permission is the same as the Linux filesystem or HDFS system.